README.md

```diff
@@ -122,6 +122,15 @@ app/views/passwordless/sessions/create.html.erb
 app/views/passwordless/mailer/magic_link.text.erb
 ```
 
+Example on create.html.erb
+```erb
+<% if @resource.present? %>
+  <p>Message</p>
+<% else %>
+  <p>Other Message</p>
+<% end %>
+```
```

Comment:
How about

```
If you'd like to let the user know whether or not a record was found, `@resource` is provided to the view. You may override `app/views/passwordless/session/create.html.erb` for example like so:

```erb
<% if @resource %>
  User found, check your inbox
<% else %>
  No user found with the provided email address?
<% end %>
```

Reply:
I changed it

Reply:
I meant to include the first line too instead of “example create.erb”

Reply:
Sorry didn't notice. Not I changed it